### DISCUSSION TITLE
pod install 에러

### DISCUSSION THREAD

**Discussion by lemonit96** [velopert/dealing-with-react-native#13] [Q&A] [answered]
질문은 또 올리게되네요....ㅎㅎ p.57에서 use_flipper!()주석 처리후  pod install 명령을 실행하니 다음과 같은 에러가 발생했는데 원인을 모르겠네요.. 알려주시면 감사하겠습니다.

![스크린샷 2021-12-30 오후 10 37 35(2)](https://user-images.githubusercontent.com/70015264/147756950-11e3b82c-96ea-4aaf-bbb2-918ab519ba28.png)

![스크린샷 2021-12-30 오후 10 37 43(2)](https://user-images.githubusercontent.com/70015264/147756958-c64777c1-5cb3-48ca-80ce-6c5230bc91b4.png)

**Comment by velopert** [comment, marked as answer]
늦게 답변하게 되어 죄송합니다.

https://philip1994.tistory.com/114

여기 해결책이 있습니다.
